single_tech_samples/streamanalytics/containers.bicep

```diff
@@ -1,20 +1,22 @@
 param name string
 param env string
+param location string
 
+var replaceName = replace('${name}', '-', '')
 resource storageAccount 'Microsoft.Storage/storageAccounts@2020-08-01-preview' = {
-  name: replace('st${name}${env}', '-', '')
-  location: resourceGroup().location
+  name: 'st${substring(replaceName, 0, min(length(replaceName), 22 - length(env)))}${env}'
```

Comment:
__This an outdated change__. The goal was to 

1. create a string within the length limit for a storage account name.
2. which also preserved the prefix and environment.

After our discussion, we decided it may be too confusing for the sample consumer